src/mono/CMakeLists.txt

```diff
@@ -392,6 +392,8 @@ elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "i686" OR CMAKE_SYSTEM_PROCESSOR STREQUAL
   set(CMAKE_SYSTEM_PROCESSOR "x86")
 elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
   set(CMAKE_SYSTEM_PROCESSOR "arm64")
+elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "loongarch64" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "LOONGARCH64")
```

Comment:
Also add https://github.com/dotnet/runtime/blob/4f9cd4d96e42ee36794480f026c8cad62b6a2407/src/mono/mono.proj#L457-L461 to enable LA64 mono cross-build.